### ISSUE TITLE
kyma provision gardener fails with unsupported gardenlinux version

### ISSUE THREAD

**Issue by suleymanakbas91** [kyma-project/cli#818]
**Description**

Provisioning a Gardener cluster using CLI fails:

```bash 
➜  ~ kyma provision gardener gcp -c kubeconfig.yaml -n sa-test -s test -p test
X Provisioning Gardener cluster on GCP
Error: unable to provision gardener cluster:
Error: shoots.core.gardener.cloud "sa-test" is forbidden: [spec.provider.workers[0].machine.image: Unsupported value: core.ShootMachineImage{Name:"gardenlinux", ProviderConfig:(*runtime.RawExtension)(nil), Version:"27.1.0"}: supported values: "machineImage(gardenlinux:184.0.0)"]

  on .kyma/clusters/gardener/berlin/sa-test-93/terraform.tf line 51, in resource "gardener_shoot" "gardener_cluster":
  51: resource "gardener_shoot" "gardener_cluster" {

```

**Workaround**

Use the extra flag to override the gardenlinux version:

`kyma provision gardener gcp -c kubeconfig.yaml -n sa-test -s test -p test -e "machine_image_version=184.0.0"`

**Comment by suleymanakbas91** [comment]
This is already fixed on the master branch and will be released with v1.27.

**Comment by tobiscr** [comment]
Release in 1.21
